samples/cellular/lwm2m_client/prj.conf

```diff
@@ -59,7 +59,6 @@ CONFIG_STREAM_FLASH=y
 
 # LTE link control
 CONFIG_LTE_LINK_CONTROL=y
```

Comment:
@SeppoTakalo @jheiskan81 The LwM2M client sample has a separate overlay for NB-IoT. If LTE-M has not been available, NB-IoT has eventually been tried anyway, even without the overlay (because of `CONFIG_LTE_NETWORK_USE_FALLBACK`). The question is, do we want to keep LTE-M and NB-IoT separate, or should we remove the NB-IoT overlay and enable both LTE-M and NB-IoT in the default configuration as I have done now? If we want to support both with the same configuration, `CONFIG_COAP_INIT_ACK_TIMEOUT_MS` needs to be increased.

Reply:
I restored the LwM2M client configuration back to as it was. The default configuration is LTE-M only and there's an overlay to switch to NB-IoT. As discussed with @jheiskan81, implementation of dynamic ack timeout is ongoing, after which there's no longer need to use different configurations for LTE-M and NB-IoT.